gridstatus/decorators.py

```diff
@@ -88,9 +89,34 @@ def wrapped_f(*args, **kwargs):
             if args_dict["date"] == "latest":
                 return f(*args, **kwargs)
 
+            default_timezone = args_dict["self"].default_timezone
+
+            # For today with hourly data, create a range that spans the day
+            if self.frequency == "HOUR_START":
+                date = args_dict.get("date")
+                if date == "today":
+                    args_dict["date"] = pd.Timestamp.now(tz=default_timezone).floor("D")
+                    args_dict["end"] = pd.Timestamp.now(tz=default_timezone).ceil("D")
+                # If date is a string or datetime.date, then we floor it to the
+                # beginning of the date. If the end date is not set, we set it to
+                # the start of the next day
+                elif isinstance(date, str) or isinstance(date, datetime.date):
+
+                    args_dict["date"] = pd.Timestamp(
+                        date,
+                        tz=default_timezone if not date.tzinfo else None,
```

Comment:
running locally im getting errors like this 
<img width="1076" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/c912e003-d7a7-4ac9-aca1-faf9607c3630">

is it possible to avoid adding new date parsing logic? can `_handle_date` do it?

Reply:
Yep, you're right, it can be handled by `_handle_date`.